SwiftWisdom/Rx/Rx+Extensions.swift

```diff
@@ -69,3 +69,9 @@ public func <-> <T>(property: ControlProperty<T>, variable: Variable<T>) -> Disp
     
     return Disposables.create(bindToUIDisposable, bindToVariable)
 }
+
+extension ObservableType {
```

Comment:
This might deserve a comment. The only reason for its existence is to continue using trailing closures, which you wouldn't be able to do with the underlying `subscribe(onNext: on)`